packages/astro/src/default/components/TutorialContent.astro

```diff
@@ -11,11 +11,18 @@ const { lesson } = Astro.props;
 const { Markdown, prev, next } = lesson;
 ---
 
-<div class="h-full overflow-auto scrollbar-transparent p-8">
+<div class="h-full overflow-auto scrollbar-transparent p-6 sm:p-8">
   <div class="markdown-content text-tk-elements-content-textColor">
     <Markdown />
   </div>
-  <div class="grid grid-cols-[1fr_1fr] gap-4 mt-8">
+  {
+    next && (
+      <a href={next?.href} class="sm:hidden underline mt-8 font-bold block text-tk-elements-navCard-textColor">
+        {lesson.data.i18n!.nextLessonPrefix} {next.title}
+      </a>
+    )
+  }
```

Comment:
So on mobile the default prev/next are hidden, and there's only next-button? Was this part of the requested improvements?

Reply:
Reverted those changes